src/render/text.rs

```diff
@@ -518,18 +550,28 @@ pub fn fill_fonts(group: &Group, ctx: &mut Context, fontdb: &fontdb::Database) {
                             font.glyph_set.insert(g.id.0, g.text.clone());
                             font.glyph_remapper.remap(g.id.0);
                         }
+
+                        if ctx.options.pdfa && g.id.0 == 0 {
+                            return Err(ConversionError::MissingGlyphs);
+                        }
```

Comment:
I don't remember, does PDF/A just talk about producing glyphs that are explicitly mapped to .notdef, or also implicitly (i.e. referencing a glyph that doesn't exist in the font)?

Reply:
can that even happen?

Reply:
Fair point, I guess it can't assuming that usvg doesn't mess something up. :p